clients/src/main/java/org/apache/kafka/clients/consumer/ShareConsumerConfig.java

```diff
@@ -26,7 +26,7 @@
 /**
  * The consumer configuration behavior specific to share groups.
  */
-class ShareConsumerConfig extends ConsumerConfig {
+public class ShareConsumerConfig extends ConsumerConfig {
```

Comment:
This should not be a public class. It's not part of the public interface. I know that it's currently referenced in `ShareConsumerImplTest` which is in the internals package, but I don't think that reference is necessary.